test/scripts/gateway/install-gateway.sh

```
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
```

I general would like to see us standardize on fail fast with bash scripts. WDYT?

```suggestion
set -eo pipefail
```

---

sure, added